### ISSUE TITLE
Mantine css not work in product version after update nextjs 14.0.1 to 14.3.2

### ISSUE THREAD

**Issue by eurus1** [mantinedev/mantine#6271]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.9.2

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

All

### Describe the bug

i use nextjs 14.1.3 and mantine componnents in my project and project work fine, but after update my nextjs version to 14.2.3 my styles not work in build wersion but when i use the dev mod everyting is ok. can help me to fix this issue?

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rosaldo260** [comment]
Yeah, I see the same issue that mantine styles over-ride my custom styles and it worked before with the same setup.

I've not found a fix for it yet.

**Comment by rtivital** [comment]
What exactly does not work?

If you need to override styles, see https://mantine.dev/styles/mantine-styles/#css-layers
